test/terraform/permanent/main.tf

```diff
@@ -1,13 +1,25 @@
-# read default vpc and create security group to validate basic setup and role permissions
-data "aws_vpc" "example_read_resource" {
+data "aws_vpc" "this" {
   id = "vpc-015d2f927c8b5dea7"
 }
 
-resource "aws_security_group" "example_write_resource" {
-  name   = "example_write_resource_from_terraform"
-  vpc_id = data.aws_vpc.example_read_resource.id
+data "aws_subnets" "this" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.this.id]
+  }
 
-  tags = {
-    created_by = "terraform"
+  filter {
+    name = "availability-zone"
+    values = ["us-east-1a", "us-east-1b"]
```

Comment:
As far as I can tell these subnets are public (ingress and egress) while we probably only need egress. I'm wondering if we should create private subnets (no internet ingress b/c all telemetry generating apps will run within the cluster) and route internet egress (to NR) through a NAT gateway. We could also separate this out into a dedicated ticket. I admit that my network knowledge/skills are limited, so I might be oversimplifying things/effort.

Reply:
Good thought, I we may need to look at what level of connectivity we'll need with GH in order to orchestrate the cluster.

Reply:
Looks like the recommendation is private, i assume the eks endpoint is separate so the above comment isn't relevant.  I'll add a couple private subnets in different zones.